src/syntax/syntax.rkt

```diff
@@ -162,104 +259,91 @@
                                             (lambda () (*unknown-d-ops* (cons 'operator (*unknown-d-ops*))) fallback)))
            (define float-proc (get-ffi-obj 'id_f #f (_fun #,@(build-list num-args (λ (_) #'_float)) -> _float)
                                            (lambda () (*unknown-f-ops* (cons 'operator (*unknown-f-ops*))) fallback)))
-           (define-operator (operator #,@(build-list num-args (λ (_) #'real))) real
-             [fl (λ args (apply (if (flag-set? 'precision 'double) double-proc float-proc) args))]
-             [key value] ...)))]))
+           (define-operator (#,(sym-append op* '.f64) #,@(build-list num-args (λ (_) #'binary64))) binary64
+             [fl (λ args (apply double-proc args))]
+             [key value] ...)
+           (define-operator (#,(sym-append op* '.f32) #,@(build-list num-args (λ (_) #'binary32))) binary32
+             [fl (λ args (apply float-proc args))]
+             [key value] ...)
+           (declare-parametric-operator/libm (syntax-e #'operator) #,num-args)
```

Comment:
Perhaps, now that every operator is parametric, we should build `declare-parametric-operator` into `define-operator`?

Reply:
Actually, there are still 3 operators that are not parametric: `or`, `and`, `not` (and `if` in some contexts). We should talk about this for the next PR.

Reply:
Interesting. `if` is sometimes treated as syntax, because it can change types as well as representations. In any case, `or` could be a parametric operator that desugars into `or`. We can make parametric operators the default now that it's almost all of them.